aviatrix/resource_aviatrix_edge_spoke_transit_attachment.go

```diff
@@ -111,6 +111,11 @@ func resourceAviatrixEdgeSpokeTransitAttachment() *schema.Resource {
 					Type: schema.TypeString,
 				},
 			},
+			"dst_wan_interfaces": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Destination WAN interface for edge gateways where the peering terminates, comma separated",
```

Comment:
Why is `dst_wan_interfaces` a single string, but using a commas as separation? Shouldn't we use a set? or some sort of list?

If API accepts input as a "string" separated by commas (eg. "a, b, c"), TF should at least follow best practices to utilize the correct data structure and marshal that data into the 'proper' output for API to consume.

Reply:
dst_wan_interfaces is accepting a single interface value. I have updated the description and documentation to reflect that change.